src/helpers/dom.coffee

```diff
@@ -10,4 +10,12 @@ module.exports = {
     if @matches(el, selector) then el else @closest(el.parentNode, selector)
 
 
+  readBootstrapData: (root = document) ->
+    el = root.querySelector('#tutor-boostrap-data')
```

Comment:
Well it is a `<script>` tag, but it's holding json.    :grin: 

No real reason, other than a slight aversion to creating a global variable.  What I really wanted to accomplish was to create an optional variable so the app would continue to function fine if it was missing for some reason.

We could just as easily create a `window.TUTOR_BOOTSTRAP_DATA` and read that.